app/api/currentuser/search_for_available_groups.robustness.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Search for groups available to the current user - robustness
+  Background:
+    Given the database has the following table 'users':
+      | ID | sLogin | tempUser | idGroupSelf | idGroupOwned | sFirstName  | sLastName | iGrade |
+      | 1  | owner  | 0        | 21          | 22           | Jean-Michel | Blanquer  | 3      |
+
+  Scenario: Should fail if the search string is not present
+    Given I am the user with ID "1"
+    When I send a GET request to "/current-user/available-groups"
+    Then the response code should be 400
+    And the response error message should contain "Missing search"
+
+  Scenario: Should fail if the search string is too small
+    Given I am the user with ID "1"
+    When I send a GET request to "/current-user/available-groups?search=%20%20%E4%B8%AD%E5%9B%BD%20%20"
```

Comment:
You should put a comment next on this line with the search you are doing so that the reader can understand the test immediately.